src/constants/demographics.ts

```diff
@@ -18,6 +18,7 @@ export const dialects: Array<Demographic> = [
     { id: 'rn-rl-framburdur', name: 'rn/rl-framburður' },
 ];
 
+// TODO: add translation for under 18
 export const ages: Array<Demographic> = [
     { id: 'barn', name: 'undir 18' },
```

Comment:
"Less than 18"?

Reply:
Todo since I need to make a specific change for it. It is either to add all ages to the translation files even though all but under 18 is just numbers. And if I just add {id: 'barn', name: 'ages.under18'} and use the translate function on it, then I'll have to make sure to not use it on the others. 

Dunno what is the cleanest solution, or even a relatively clean one. Could also just check, if language is english, then replace 'i' with 'e' but that feels ugly